### PR TITLE
[Thorvg] Update Thorvg port to v0.11.0

### PR DESCRIPTION
**Pull request by rinechran**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/thorvg/portfile.cmake [hunk 1]
```diff
@@ -16,6 +16,9 @@ else()
 endif()
 
 if ("tools" IN_LIST FEATURES)
+    if(VCPKG_TARGET_IS_WINDOWS)
```
Comment (BillyONeal) [inline ports/thorvg/portfile.cmake:19]: Why did you change this from the `supports` expression? Telling people up front something doesn't work (as supports does) is a better experience than building dependencies and stuff only for it to explode later..

**Review by BillyONeal — CHANGES_REQUESTED** [review]

File: ports/thorvg/portfile.cmake (hunk 1 above)
Comment (rinechran) [reply ports/thorvg/portfile.cmake:19]: That's a good sound

Comment (BillyONeal) [reply ports/thorvg/portfile.cmake:19]: This doesn't look changed

Comment (rinechran) [reply ports/thorvg/portfile.cmake:19]: I'm sorry. I think I made a mistake I'll put up the PR again

**Review by BillyONeal — CHANGES_REQUESTED** [review]
The supports expression removal is still here :(

**Comment by rinechran** [comment]
> The supports expression removal is still here :(

Is the talking support by any chance vcpkg.json's support?

**Comment by BillyONeal** [comment]
> > The supports expression removal is still here :(
> 
> Is the talking support by any chance vcpkg.json's support?

There are still changes here which remove a `"supports"` from `vcpkg.json` and add an equivalent block in `portfile.cmake`. Above it looks like you said you would revert that change but it's still here.

**Comment by eunsikNota** [comment]
> > > The supports expression removal is still here :(
> > 
> > 
> > Is the talking support by any chance vcpkg.json's support?
> 
> There are still changes here which remove a `"supports"` from `vcpkg.json` and add an equivalent block in `portfile.cmake`. Above it looks like you said you would revert that change but it's still here.

Thank you for understanding me

**Comment by rinechran** [comment]
> > > The supports expression removal is still here :(
> > 
> > 
> > Is the talking support by any chance vcpkg.json's support?
> 
> There are still changes here which remove a `"supports"` from `vcpkg.json` and add an equivalent block in `portfile.cmake`. Above it looks like you said you would revert that change but it's still here.

Thank you for understanding me

**Review by BillyONeal — APPROVED** [review]
Thanks for the update!

**Comment by rinechran** [comment]
I didn't know it changed a lot because I contributed after a long time.  Thank you so much for letting me know. @BillyONeal

**Pull request merged**